assets/js/custom/Translation.js

```diff
@@ -17,11 +122,6 @@ function Translation (hasDescription, hasCredit) {
     document.documentElement.lang
   )
 
-  $('.comment-translate-button').each(function () {
-    const commentId = $(this).attr('id').substring('comment-translate-button-'.length)
-    translation(this, document.getElementById('comment-text-' + commentId), document.documentElement.lang)
-  })
-
   function translation (buttonElement, textElements, targetLang) {

```

Comment:
maybe this can be renamed to something that makes more sense now and that differentiates with the new approach?

Reply:
Renamed to translateWithLink